index.bs

```diff
@@ -41,9 +41,6 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
 		type: dfn; text: relevant settings object
 		url: #event-loop-processing-model; type: dfn; text: HTML Processing Model
 		url: #concept-environment-top-level-origin; type: dfn; text: top-level origin
-	urlPrefix: infrastructure.html;
-		type: dfn; text: rules for parsing dimension values
```

Comment:
Removing the reference to `rules for parsing dimension values` is probably not intended?

Reply:
That reference is no longer used in the spec, this is just cleanup that snuck into this patch. I can remove it if you'd prefer a clean patch.

Reply:
Either way is fine. Maybe note it in the commit message?

Reply:
Will do.

Reply:
Done.